Classes/Tools.php

```diff
@@ -172,14 +182,14 @@ public static function runSpool(Newsletter $limitNewsletter = null)
         $emailSentCount = 0;
         $mailers = [];
 
-        $objectManager = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Extbase\Object\ObjectManager::class);
-        $newsletterRepository = $objectManager->get(\Ecodev\Newsletter\Domain\Repository\NewsletterRepository::class);
-        $emailRepository = $objectManager->get(\Ecodev\Newsletter\Domain\Repository\EmailRepository::class);
+        $newsletterRepository = self::getNewsletterRepository();
+        $emailRepository = self::getObjectManager()->get(\Ecodev\Newsletter\Domain\Repository\EmailRepository::class);
 
         $allUids = $newsletterRepository->findAllNewsletterAndEmailUidToSend($limitNewsletter);
 
         $oldNewsletterUid = null;
         foreach ($allUids as $uids) {
+            $newsletter = null;
```

Comment:
This will break the sending to a single Newsletter with multiple language. In that case the sending will crash because `$newsletter` will be `null` and the call to `getConfiguredMailer` will fail.